manager/integration/tests/test_basic.py

```diff
@@ -3252,45 +3252,233 @@ def test_multiple_volumes_creation_with_degraded_availability():
+    client.update(node2, allowScheduling=True)
+
+    # wait to complete restore
+    common.wait_for_volume_restoration_completed(client, dst_vol_name)
+    dst_vol = common.wait_for_volume_detached(client, dst_vol_name)
+    assert dst_vol.conditions[VOLUME_CONDITION_SCHEDULED]['status'] == "True"
+
+    # attach the volume
+    create_pv_for_volume(client, core_api, dst_vol, dst_vol_name)
+    create_pvc_for_volume(client, core_api, dst_vol, dst_vol_name)
+
+    dst_pod_name = dst_vol_name + "-pod"
+    pod['metadata']['name'] = dst_vol_name + "-pod"
+    pod['spec']['volumes'] = [{
+        'name': pod['spec']['containers'][0]['volumeMounts'][0]['name'],
+        'persistentVolumeClaim': {
+            'claimName': dst_vol_name,
+        },
+    }]
+    create_and_wait_pod(core_api, pod)
+    # check 2 replica scheduled successfully
+    dst_vol = common.wait_for_replica_scheduled(client, dst_vol_name,
+                                                to_nodes=[node1.name,
+                                                          node2.name],
+                                                expect_success=2,
+                                                expect_fail=1,
+                                                chk_vol_healthy=False,
+                                                chk_replica_running=False)
+    wait_for_volume_condition_scheduled(client, dst_vol_name,
+                                        "status", "False")
+
+    # verify the data
+    dst_md5sum = get_pod_data_md5sum(core_api, dst_pod_name, data_path)
+    assert src_md5sum == dst_md5sum
+
+
+def test_allow_volume_creation_with_degraded_availability_dr(
+        set_random_backupstore, client, core_api, volume_name, csi_pv, pvc, pod, pod_make):  # NOQA
+    """
+    Test Allow Volume Creation with Degraded Availability (Restore)
 
+    Requirement:
+    1. Set `allow-volume-creation-with-degraded-availability` to true.
+    2. `node-level-soft-anti-affinity` to false.
+    3. Create a backup of 800MB.
+
+    Steps:
     (DR volume)
-    1. Disable scheduling for node 2 and 3
-    2. Create a DR volume from backup with three replicas.
-        1. Volume should be attached automatically and `Scheduled` is true
-        2. One replica schedule succeed. Two other replicas failed scheduling.
-    3. During the restore, enable scheduling for node 2.
-        1. One additional replica of the volume will become scheduled
-        2. The other replica is still failed to schedule.
-        3. Scheduled condition is still true
-    4. Wait for the restore to complete.
-    5. Enable the scheduling for node 3.
-        1. DR volume should automatically rebuild the third replica.
-    6. Activate the volume and verify the data.
+    1. Disable scheduling for node 2 and 3.
+    2. Create a DR volume from backup with 3 replicas.
+        1. The scheduled condition is false.
+        2. Only node 1 replica become scheduled.
+    3. Enable scheduling for node 2 and 3.
```

Comment:
We can wait for all replica scheduling then restore/rebuild success here since DR volume won't be detached automatically if it's not activated

Reply:
Addressed.